src/components/Card/IndicatorCard/IndicatorCard.js

```diff
@@ -46,24 +103,31 @@ const IndicatorCard = (props) => {
     >
       <p className={style.header}>{indicator.name}</p>
 
-      {!indicator.contextual && (
+      {!props.isContextPage && (
```

Comment:
Is this conditional necessary? I thought we wanted icons to show everywhere?

Reply:
We do, but the decision to strip out the logic of not showing it on the context page came after I'd finished. And TBH strikes me as a decision with 40% chance of mind changing.